.github/workflows/publish-and-deploy-gamma.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish and Deploy (Gamma)
+permissions: read-all
+
+on:
+  push:
+    branches: [ dev ]
```

Comment:
Gamma branch for the gamma env?

Reply:
In either case, needs a branch protection rule + env deployment (to branch) restriction

Reply:
We were going to switch gamma and dev eventually. I gotta remember how exactly we wanted to go about that.